<launch>
    <arg name="camera" default="camera" />
    <group ns="$(arg camera)">
        <node pkg="astra_camera" type="camera_node" name="uvc">
            <param name="index" value="0"/>

            <param name="width" value="640"/>
            <param name="height" value="480"/>
            <param name="video_mode" value="mjpeg"/>
            <param name="frame_rate" value="15"/>

            <param name="timestamp_method" value="start"/>
            <param name="camera_info_url" value=""/>

            <remap from="/$(arg camera)/image_raw" to="/$(arg camera)/rgb/image_raw" />
        </node>
    </group>
    <node pkg="image_transport" type="republish"
          args="raw in:=/$(arg camera)/rgb/image_raw compressed out:=/$(arg camera)/rgb/image_rect_color"
          name="compressed_pub" output="screen"/>
    <node pkg="face_recognition_ultra_light" type="face_detection.py" name="face_detection" output="screen" />
    <node pkg="face_recognition_ultra_light" type="emotion_recognition.py" name="emotion_recognition" output="screen" />
</launch>